.github/CONTRIBUTING.md

```diff
@@ -1,13 +1,22 @@
 # Contributing
+![](https://raw.githubusercontent.com/checkstyle/resources/master/img/checkstyle-logos/checkstyle-logo-260x99.png)
 
-Please see instructions for contributing here:
-https://checkstyle.org/contributing.html
+#### [Reporting issues](https://checkstyle.org/report_issue.html)
```

Comment:
Is there a reason for this item when we have the similar link on line 7?

Reply:
@varunu28 Still no answer on this.

Reply:
@rnveach I think its the same template on both the links. I can remove the template on Github link and just keep the link to the checkstyle website

@romani Please confirm

Reply:
I am ok with small duplication, it is hard to force users to follow issue template, the more reference to such page the better

Reply:
I will mark this as resolved then.